dapps/frenemies/src/routes/Home.tsx

```diff
@@ -1,6 +1,10 @@
 // Copyright (c) Mysten Labs, Inc.
 // SPDX-License-Identifier: Apache-2.0
 
+import Layout from './../components/Layout';
+
 export function Home() {
-  return <h1 className="text-3xl font-bold underline">home page</h1>;
+  return (
+    <Layout />
```

Comment:
Rather than rendering this here, you should throw this in the `Root` component, which wraps these components. Just have it take a `children` prop, and then you can use it in Root like `<Layout><Outlet /></Layout>`